pages/common/hg.md

```diff
@@ -0,0 +1,19 @@
+# hg
+
+> Mercurial distributed source control management.
```

Comment:
This feels like it could be a be more descriptive, maybe?

Reply:
Maybe something such as?
> The `hg` command provides an interface for Mercurial, a distributed source control management system.